pkg/kubectl/stop.go

```diff
@@ -509,3 +507,7 @@ func (reaper *ServiceReaper) Stop(namespace, name string, timeout time.Duration,
 	}
 	return services.Delete(name)
 }
+
+func getNonOrphanDeleteOptions() *api.DeleteOptions {
+	return &api.DeleteOptions{OrphanDependents: new(bool)}
```

Comment:
`&api.DeleteOptions{OrphanDependents: false}` won't work because the boolean passed to `OrphanDependents` needs to be a pointer. Hence, the use of `new`.